model/task/db.go

```diff
@@ -188,17 +188,17 @@ var (
 		},
 	}
 
-	updateDisplayTasksAndTasksExpression = bson.M{
-		"$set": bson.M{
-			CanResetKey: true,
-		},
-		"$unset": bson.M{
-			AbortedKey:              "",
-			AbortInfoKey:            "",
-			OverrideDependenciesKey: "",
+	updateDisplayTasksAndTasksSet = bson.M{"$set": bson.M{
```

Comment:
could we change the formatting back? leaving white space between the bson statements is our typical formatting; it simplifies skimming the expression to see what changes are being made

Reply:
(i'm gonna reply to your top-level comment here so we can have a thread)

> Since we aren't using the value yet, caching it in stages would've been okay

Unfortunately this is not quite true—since `GetDisplayStatus` checks to see if the field is defined, once DisplayStatus is set by any function it will begin to be used. This means that having it set inconsistently would be bad.

But as I type this, I realize another approach could have been to use a new field name in these updates to avoid that problem. Honestly if this feels too high-risk to you I'd be willing to do move to that approach which could permit making these updates incrementally.